Electron/Core/RoveProtocol/Rovecomm.ts

```diff
@@ -67,7 +67,7 @@ function parse(packet: Buffer): void {
    *   0                   1                   2                   3
    *   0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
    *  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-   *  |    Version    |            Data Id            |  Data  Count  |
+   *  |    Version    |            Data Id            |  Data Count  |
```

Comment:
The double space here and on line 251 is necessary to reflect the size of data count